zadanie zrobione

jtermi opened the pull request.

jtermi committed:
zadanie zrobione

jtermi committed:
zadanie zrobione

slawekludwiczak commented on src/main/java/pl/javastart/task/Print.java:
```diff
@@ -0,0 +1,9 @@
+
+public class Print {
+
+    void roomIformation(Room room) {
```
*information
W nazwach metod używamy też czasowników, czyli np. get, print itp.

slawekludwiczak commented on src/main/java/pl/javastart/task/Room.java:
```diff
@@ -0,0 +1,53 @@
+    }
+
+    boolean lowerTemperature() {
+        if (airConditioned && roomTemperature > minTemperature) {
+            roomTemperature--;
```
Jeżeli `roomTemperature` będzie np. 19.5, a minTemperature to 19, to temperatura spadnie do 18.5, a powinna się zatrzymać na 19.

jtermi committed:
poprawione błędy
- nazewnictwo metody wyświetlającej informacje o pokoju
- logika metody obniżającej temperaturę
  zamiast korygowania temperatury po pierwotnym obniżeniu poniżej minium obniżana jest po poprawce tylko i wyłącznie do poziomu minimum.
